### DISCUSSION TITLE
Quick help with assist formatting??

### DISCUSSION THREAD

**Discussion by rkierstead8** [dbeaver/dbeaver#12084] [Q&A]
I recently Upgraded from 7.0 and this is a real thorn in my side.

In the older version , when choosing a specific package, it would always populate the expected parameters , but now it only gives me the format and not the name of what I should be passing in.

Example:

example.NEW__(:Part_no_, :User_ID_) 

example.NEW__(VARCHAR2, VARCHAR2) 

I noticed this shortly after 7.0 and have been reluctant to upgrade because of it, but ever once in a while, I try it to see if my issue is "fixed", but it seems that this is intentional and won't be going back to the way I like it. Is there a setting somewhere I can change or do I just need to revert to 7.0 indefinitely?

**Comment by LonwoLonwo** [comment]
Hello @rkierstead8 
What database do you use?
Can you provide DDL of your package for testing, please?
And examples of all query, where you use autocomplete for packages?
Because autocomplete parameters depend oon part of the query (after CREATE word, or after SELECT etc.)
